apps/core-app/src/utils/contextHelpers.ts

```diff
@@ -83,11 +124,10 @@ export const loadMembersList = async ({
     });
 
     if (shouldUpdate) {
-      if (res.nextPaging) {
-        setNextPaging(res.nextPaging);
-      }
+      setNextPaging(res.nextPaging);
 
       setData((prevState) => {
+        if (deepEqual(prevState, res.items)) return res.items;
```

Comment:
Is this the pattern we're using to deal with the useEffect issue?

Reply:
> Is this the pattern we're using to deal with the useEffect issue?

Not really - just preventing updating state again. I can't figure out the double use effect firing issue in development/strict mode